### PR TITLE
Fix eslint issues in the sandbox editor.

### PR DESCRIPTION
**Pull request by lbogdan**
Fixes https://github.com/CompuIves/codesandbox-client/issues/142.

### CODE REVIEW

File: src/app/components/sandbox/CodeEditor/CodeMirror.js [hunk 1]
```diff
@@ -410,8 +410,9 @@ export default class CodeEditor extends React.PureComponent<Props, State> {
     }
 
     if (preferences.lintEnabled) {
+      const initialized = 'eslint' in window;
       System.import('app/utils/codemirror/eslint-lint')
-        .then(initializer => initializer.default())
+        .then(initializer => !initialized && initializer.default())
```
Comment (lbogdan) [inline src/app/components/sandbox/CodeEditor/CodeMirror.js]: `eslint-lint.js`.`initialize()` (the default export) should execute only when importing eslint the first time. This gets triggered whenever switching between CodeMirror and Monaco.

File: src/app/utils/codemirror/eslint-lint.js [hunk 2]
```diff
@@ -377,6 +382,7 @@ export default (async function initialize() {
     await delay(100);
   }
 
+  eslintRequire = window.require;
```
Comment (lbogdan) [inline src/app/utils/codemirror/eslint-lint.js]: Save eslint's require.

File: src/app/utils/codemirror/eslint-lint.js [hunk 3]
```diff
@@ -336,7 +337,11 @@ function getSeverity(error) {
 
 function eslintValidate(text) {
   if (!linter) return [];
-  return linter.verify(text, defaultConfig);
+  const saveRequire = window.require;
+  window.require = eslintRequire;
+  const result = linter.verify(text, defaultConfig);
+  window.require = saveRequire;
+  return result;
```
Comment (lbogdan) [inline src/app/utils/codemirror/eslint-lint.js]: When switching from Monaco to CodeEditor, `window.require` refers to Monaco's loader, clashing with `linter.verify()` which calls `require()` for some reason. So we save eslint's require (see below) and use it every time we call `linter.verify()`.

**Comment by CompuIves** [comment]
GREAT! Thanks!! 👍

**Pull request merged**